.github/pull_request_template.md

```diff
@@ -6,4 +6,4 @@
 - Commit messages and codestyle follow [conventions](./CONTRIBUTING.md).
 - Relevant issues are linked in the PR description.
 - Tests added for new functionality.
-- Documentation/comments updated according to changes.
```

Comment:
`end-of-file-fixer` consistently ends files with a line break. This is probably what happened here